dist/src/assembly/bin/zeneventserver-functions.sh

```diff
@@ -31,6 +33,8 @@ run() {
 }
 
 run_quiet() {
+    update_schema_in_background
```

Comment:
Why upgrade in the background?  Wouldn't we want the upgrade finished before ZEP starts, especially if it is a long running DB change?